sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/ClassicTokenizerConverter.java

```diff
@@ -37,7 +37,7 @@ public static ClassicTokenizer map(com.azure.search.documents.indexes.implementa
 
         Integer maxTokenLength = obj.getMaxTokenLength();
         classicTokenizer.setMaxTokenLength(maxTokenLength);
-        classicTokenizer.validate();
+
```

Comment:
Just curious, if these were autorest updates for client-side validations that were removed?

Reply:
Yes, client side validation generation was removed.